ecosystem/python/sdk/aptos_sdk/bcs.py

```diff
@@ -155,6 +155,11 @@ def map(
             self.fixed_bytes(key)
             self.fixed_bytes(value)
 
+    def sequence_serializer(
+        value_encoder: typing.Callable[[Serializer, typing.Any], bytes],
+    ):
+        return lambda self, values: self.sequence(values, value_encoder)
+
     def sequence(
         self,
```

Comment:
Interesting, is this so you can delay serialization?

Reply:
Yeah, we use this for specifying how to serialize a transaction argument in a fancy way... I am a fancy guy after all